fuel-forge/src/main/java/com/github/kittinunf/fuel/forge/FuelForge.kt

```diff
@@ -22,14 +22,9 @@ inline fun <reified T : Any> Request.responseObject(noinline deserializer: JSON.
 
 inline fun <reified T : Any> Request.responseObjects(noinline deserializer: JSON.() -> DeserializedResult<T>) = response(forgesDeserializerOf(deserializer))
 
-inline fun <reified T : Any> forgeDeserializerOf(noinline deserializer: JSON.() -> DeserializedResult<T>) = object : ResponseDeserializable<T> {
-
+fun <T : Any> forgeDeserializerOf(deserializer: JSON.() -> DeserializedResult<T>) = object : ResponseDeserializable<T> {
```

Comment:
Maybe we mark this as private as we don't want our client to call this directly.

Reply:
If I marked it as private, I think I will break the test.

Reply:
This does feel like we have removed the `inline` modifier which will remove any performance enhancements and it makes the jacoco diff look better, however if this pr contains no more unit test or assertions, in which case how have we really improved the coverage?